portfolio/src/main/webapp/index.html

```diff
@@ -20,50 +40,43 @@
-      </section>
+
+      <div id="overlay">
+        <div class="layer layer-left"></div>
+        <div class="layer layer-middle"></div>
+        <div class="layer layer-right"></div>
+      </div>
 
       <section id="about-me-section" class="white-background">
-        <h1 class="section-header">About me</h1>
+        <h1 id="about-me-title" class="section-header"><span>About Me</span></h1>
         <div id="about-me-content" class="section-content">
-          <div id="bio-content">
-            <div id="selfie-container">
-              <img id="selfie" src="./static-files/images/selfie-cropped.jpg">
-            </div>
+          <div id="selfie-container">
+            <img id="selfie" src="./static-files/images/selfie.jpg">
+          </div>
+          <div id="bio">
             <p>
-              Hello World! I'm Sabrina and I'm a software developer from Gainesville, Florida. I'm currently a raising junior at the University of Florida, pursuing a degree in Computer Science and expanding my knowledge through personal projects.
+              I'm a software developer and computer science junior at the University of Florida. I love 
+              exploring new fields of programming and using my software skills to solve real-world problems.
             </p>
-          </div>
-
-          <div id="skills-content">
-            <p class="skill-label remove-top-spacing">Java</p>
-            <div class="skillbar-outline"><div class="skillbar java-bar">90%</div></div>
-
-            <p class="skill-label">C++</p>
-            <div class="skillbar-outline"><div class="skillbar cpp-bar">80%</div></div>
-
-            <p class="skill-label">HTML</p>
-            <div class="skillbar-outline"><div class="skillbar html-bar">70%</div></div>
-
-            <p class="skill-label">CSS</p>
-            <div class="skillbar-outline"><div class="skillbar css-bar">70%</div></div>
-
-            <p class="skill-label">Javascript</p>
-            <div class="skillbar-outline"><div class="skillbar javascript-bar">50%</div></div>
-
-            <p class="skill-label">Python</p>
-            <div class="skillbar-outline"><div class="skillbar python-bar">50%</div></div>
+            <div id="skills">
+              <div class="row">
+                <span id="Java" class="programming-language" data-language="Java" data-years="5 years"></span>
+                <span id="Python" class="programming-language" data-language="Python" data-years="3 years"></span>
+                <span id="Cpp" class="programming-language" data-language="C++" data-years="2 years"></span>
+              </div>
+              <div class="row">
+                <span id="HTML" class="programming-language" data-language="HTML" data-years="3 years"></span>
+                <span id="CSS" class="programming-language" data-language="CSS" data-years="3 years"></span>
+                <span id="Javascript" class="programming-language" data-language="Javascript" data-years="1 year"></span>
```

Comment:
Are you still using the id attribute now?

Reply:
Nope. Deleted 👍 .